packages/cookbook/src/accounts/access-keys/create-full-access-key.js

```diff
@@ -0,0 +1,41 @@
+const { KeyPair, keyStores, connect } = require('near-api-js');
+const path = require('path');
+const os = require('os');
+
+async function createFullAccessKey({ accountId, keyStore, networkId, nodeUrl }) {
```

Comment:
I went with this pattern to make it easier for new developers to see the dependencies for each recipe in their simplest form. In particular extracting `keyStore` allows the local script invocation to use a file system keystore while the tests can use the in-memory implementation. I've tried to remove unnecessary dependencies where possible (e.g. removing `keyStore` for view function calls).